helm/chart-operator-chart/templates/configmap.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Values.namespace }}
+data:
+  config.yaml: |
+    server:
+      listen:
+        address: 'http://0.0.0.0:{{ .Values.port }}'
```

Comment:
I see what this is supposed to do. My question though is if this works with draughtsman. This here usually means that whatever we substitute comes from the values files draughtsman is using so all the custom magic this project here provides has to be merged with the actual draughtsman values and I am not sure if this is the case.

Reply:
We're not using `draughtsman` but we do need configuration management.

Reply:
Also helm supports including multiple `values.yaml` files and is not substituting whole files. If there would be same keys that would be a problem, but prefixing them as e.g. `draughtsman` with `Installations. ...` for external files would be a solution .